website/templates/prereg_landing_page.mako

```diff
@@ -134,18 +95,25 @@
           <table class="prereg-button-row">
             <tbody>
               <tr>
-                ## Always displayed
+                %if is_logged_in:
                 <td class="col-sm-${ num_cols } prereg-button-col">
-                  <div class="prereg-button m-b-md p-md osf-box-lt p-md box-round prereg" data-qtoggle-group="prereg" data-qtoggle-target="#newPrereg">Start a new ${kind()}</div>
+                  <div class="prereg-button prereg-button-qtoggle m-b-md p-md osf-box-lt p-md box-round prereg" data-qtoggle-group="prereg" data-qtoggle-target="#newPrereg">Start a new pregistration</div>
                 </td>
+                %else:
+                <td class="col-sm-1 prereg-button-col">
```

Comment:
Was going to include { num_cols } here but missed it. Will add if there are requested changes, but not a big enough deal since it can only ever be 1